python/sglang/lang/backend/openai.py

```diff
@@ -379,10 +379,12 @@ def openai_completion(
     for attempt in range(retries):
         try:
             if is_chat:
-                if "stop" in kwargs and kwargs["stop"] is None:
-                    kwargs.pop("stop")
+                kwargs.pop("stop", None)
```

Comment:
Could you clarify why this change is needed?

Reply:
I think these two are equivalent? If `"stop"` not in `kwargs`, `kwargs.pop("stop", None)` won't do anything.

Reply:
What if "stop" in kwargs and kwargs["stop"] is not None? (then "stop" should not be popped from `kwargs`

Reply:
You are right.